src/scripts/apploader.js

```diff
@@ -1,12 +1,24 @@
-! function() {
+(function() {
```

Comment:
Can we just unwrap the function entirely? I've always disliked this shorthand.

Reply:
I refactored this file to further simplify the logic and removed the other IIFE, but imo this one should stay to avoid polluting the browser's global scope with values defined here.